src/DDTrace/Tracer.php

```diff
@@ -399,6 +404,25 @@ private function addHostnameToRootSpan()
         }
     }
 
+    private function addUrlAsResourceNameToRootSpan()
+    {
```

Comment:
I think the only thing we miss is to make this OFF by default. This in order to prevent unexpected huge cardinality (and so bill) issues out of the blu.